### PR TITLE
Clarify `.liquid` overwrite and ignore behavior

### PR DESCRIPTION
**Pull request by gibbz00**
Fixes: #1294

### CODE REVIEW

File: guide/src/templates/ignoring.md [hunk 1]
```diff
@@ -14,3 +14,6 @@ ignore = [
 ```
 
 Both files and folders may be ignored using this method, but currently wildcards are **not supported**.
+
+Note that `cargo-generate` checks for which files to ignore after the removal of any `.liquid` file extensions.
```
Comment (sassman) [inline guide/src/templates/ignoring.md:18]: I think this way brings more clarity.

File: tests/integration/helpers/project_builder.rs [hunk 2]
```diff
@@ -95,6 +96,23 @@ impl ProjectBuilder {
             .success();
     }
 
+    fn git_commit(message: &str, current_directory: impl AsRef<std::path::Path>) {
+        Command::new("git")
+            .args([
+                "-c",
+                "user.name='Foo Bar'",
```
Comment (sassman) [inline tests/integration/helpers/project_builder.rs:103]: great idea, to inline the git user detail here, normally on CI we set this extra up, but doing it this way would make CI simpler and we can reduce more boiler plate 👍

**Review by sassman — APPROVED** [review]

File: tests/integration/helpers/project_builder.rs (hunk 2 above)
Comment (gibbz00) [reply tests/integration/helpers/project_builder.rs:103]: Yeah, I was unsure how to run the tests locally too without adding this 😅 my git config picks user info depending if current directory is in my "personal" or "work" directory only.

File: guide/src/templates/ignoring.md [hunk 3]
```diff
@@ -14,3 +14,6 @@ ignore = [
...
+Meaning; Setting `ignore` to `["file.txt"]` will result in the ignoring of a file named `file.txt.liquid`.
```
Comment (seffradev) [inline guide/src/templates/ignoring.md:19]: I'd say "will result" could be clearer by saying "will also result". But it's just a technicality.

**Pull request merged**